src/legacy/core_plugins/metrics/public/index.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// @ts-ignore
+import { functionsRegistry } from 'plugins/interpreter/registries';
+// @ts-ignore
+import { tsvb } from './tsvb_fn';
+// @ts-ignore
+import { MetricsVis } from './kbn_vis_type';
+import { visualizations } from '../../visualizations/public';
+
+export class Plugin {
+  // @ts-ignore
```

Comment:
Would be nice not to `// @ts-ignore` here, and instead to provide interfaces for `core` and `plugins`

Reply:
Agree. Unfortunately, I can't find suitable interfaces at `src/core/public`

Reply:
Added